go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -51,7 +51,7 @@ import (
 )
 
 const (
-	builtinBackupEngineName = "builtin"
+	BuiltinBackupEngineName = "builtin"
```

Comment:
Why do you need to export this? I see no usages outside the package.

Reply:
Huh can't remember why I did this.. will revert